README.md

```diff
@@ -10,7 +10,7 @@
 
 ## Evaluator functions
 
-### `.setScores(uint roundIndex, address payable[] addresses, uint[] scores, bool moreScoresExpected)`
+### `.setScores(uint roundIndex, address payable[] addresses, uint[] scores)`
```

Comment:
Should we document that:
1. scores must eventually add up to exactly MAX_SCORE
2. if they don't add up yet, then we consider the evaluation to be incomplete and expect more calls to `setScores`


Paraphrasing my Slack message:

> We can tell whether more scores are expected or not by summing up scores submitted so far and checking if they are exactly MAX_SCORE:
> - If it’s less than MAX_SCORE, then more scores are expected
> - If it’s exactly MAX_SCORE, then we are done
> - If it’s more than MAX_SCORE, then the scores are invalid

Reply:
good idea!

90aefe3